src/crc32.c

```diff
@@ -142,10 +149,13 @@ bool generic_crc32(target_s *const target, uint32_t *const result, const uint32_
 
 bool generic_crc32(target_s *const target, uint32_t *const result, const uint32_t base, const size_t len)
 {
-	uint8_t bytes[128U];
+	uint8_t bytes[1024U]; /* ADIv5 MEM-AP AutoInc range */
 
 	CRC_CR |= CRC_CR_RESET;
 
+#if ENABLE_DEBUG == 1
```

Comment:
Might be worth using `#ifndef DEBUG_WARN_IS_NOOP` here given this is specifically for DEBUG_WARN level diagnostics. Looking at the generic implementation of this, the same suggestion applies there.

Reply:
Any loglevel works, as long as it gets compiled into BMP optionally. I'd even say it's DEBUG_INFO. Your refactor on noops will surely be idiomatic to use here, even if I don't get build errors for `all_platforms` for now. I was just extending stuff originally here, I could change DEBUG_WARN to something more appropriate now that it exists.

Reply:
Think it is technically either target-level debugging or informational - definitely not a warning. Please update to whichever of DEBUG_INFO or DEBUG_TARGET you feel is more appropriate.

Reply:
Updated to `DEBUG_INFO` and relevant NOOP macros (not used elsewhere unfortunately). IMO `DEBUG_TARGET` is for target-specific drivers, and this is part of generic BMD gdbserver core.

Reply:
You are somewhat correct about DEBUG_TARGET - the actual meaning and reason for it's introduction is target-specific/level diagnostics - like figuring out why something is running so slow against the target.. but we're happy in this case to accept the argument that this is informational and so belongs in the DEBUG_INFO level diagnostics :)